### PR TITLE
Fix tutorial example

### PR DESCRIPTION
**Pull request by oliver-sanders**
See https://github.com/cylc/cylc-flow/issues/5150

**Requirements check-list**
- [x] I have read `CONTRIBUTING.md` and added my name as a Code Contributor.

### CODE REVIEW

File: src/tutorial/scheduling/datetime-cycling.rst [hunk 1]
```diff
@@ -151,33 +151,30 @@ Recurrence Formats
    As with integer cycling, recurrences start at the :term:`initial cycle
    point` by default. We can override this in two ways:
 
-By giving an arbitrary start cycle point (``datetime/recurrence``):
+.. rubric::
+   By giving an arbitrary start cycle point (``datetime/recurrence``):
 
-.. csv-table::
```
Comment (oliver-sanders) [inline src/tutorial/scheduling/datetime-cycling.rst:156]: These tables weren't working well (horizontal scrollbar and recurrences split over two lines), switched to a regular definition list.

File: src/tutorial/scheduling/datetime-cycling.rst [hunk 2]
```diff
@@ -151,33 +151,30 @@ Recurrence Formats
...
+   By offset, relative to the initial cycle point (``offset/recurrence``).
 
-.. csv-table::
-   :header: Example, Description
-   :align: left
-   :widths: 30, 70
+The offset must be an ISO8601 duration preceded by a plus character:
 
-   ``+P1Y/P1Y``, every year starting one year after the initial cycle point.
-   ``+PT1H/T00``, "every day starting on the first midnight after the point one
```
Comment (oliver-sanders) [inline src/tutorial/scheduling/datetime-cycling.rst:179]: This was the broken recurrence.

**Review by datamel — APPROVED** [review]
This looks good to me, thanks. 

Quick separate query which I noticed after building this (although is unrelated to this pr). When you build the docs do you get the images rendering incorrectly?...
image
If this isn't just a Mel problem I can pop an issue up for it.

**Comment by oliver-sanders** [comment]
That looks like this issue - https://github.com/cylc/cylc-doc/pull/246

We worked around it by using a different version of Graphviz in the deployment actions but I don't think we have have a proper fix.

File: src/tutorial/scheduling/datetime-cycling.rst [hunk 3]
```diff
@@ -151,33 +151,30 @@ Recurrence Formats
...
-   ``2000-01-01T00/T00``, "Every day at midnight, starting on the 1st of January 2000"
-   ``2000-01-01T12/T00``, "Every day at midnight, starting on the first midnight
-   after the 1st of January at 12:00 (i.e. ``2000-01-02T00``)."
+``2000/P4Y``
+   Every fourth year, starting with the year 2000.
+``2000-01-01T00/T00``
+   Every day at midnight, starting on the 1st of January 2000.
+``2000-01-01T12/T00``
+   Every day at midnight, starting on the first midnight
+   after the 1st of January at 12:00 (i.e. ``2000-01-02T00``).
```
Comment (MetRonnie) [inline src/tutorial/scheduling/datetime-cycling.rst:163]: These are broken too?

**Review by MetRonnie — CHANGES_REQUESTED** [review]

**Comment by hjoliver** [comment]
Spellcheck error fixed by #543

**Pull request merged**